### PR TITLE
feat: Add a pancake spinner

### PR DESCRIPTION
**Pull request by RabbitDoge**
Add a spinner for pages transitions
https://www.figma.com/file/q5dKq3BuQZ4YzV5pR0Mt2M/Design-System-%2B-UI?node-id=2259%3A1

### CODE REVIEW

File: src/components/Spinner/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const Container = styled.div`
+  position: relative;
+`;
+
+const RotatingPancakeIcon = styled(PancakeIcon)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  animation: ${rotate} 2s linear infinite;
```
Comment (hachiojidev) [inline src/components/Spinner/Spinner.tsx:36]: Might want to consider making the duration customizable in the future.

File: src/components/Spinner/Spinner.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+const FloatingPanIcon = styled(PanIcon)`
+  animation: ${float} 6s ease-in-out infinite;
+  transform: translate3d(0, 0, 0);
+`;
+
+const Spinner: React.FC<SpinnerProps> = ({ size = 300 }) => {
+  return (
+    <Container>
+      <RotatingPancakeIcon width={`${size * 0.41}px`} />
```
Comment (hachiojidev) [inline src/components/Spinner/Spinner.tsx:48]: Where does .41 come from?

**Review by hachiojidev — APPROVED** [review]

File: src/components/Spinner/Spinner.tsx (hunk 2 above)
Comment (RabbitDoge) [reply src/components/Spinner/Spinner.tsx:48]: from the original image ratio

File: src/components/Spinner/Spinner.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/components/Spinner/Spinner.tsx:36]: I think the duration should be decided by the designer only

**Pull request merged**